hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/TestHoodieClientMultiWriter.java

```diff
@@ -481,6 +481,7 @@ public void testMultiWriterWithAsyncTableServicesWithConflict(HoodieTableType ta
     final SparkRDDWriteClient client3 = getHoodieWriteClient(cfg);
 
     // Create upserts, schedule cleaning, schedule compaction in parallel
+    String fourthInstantTime = HoodieActiveTimeline.createNewInstantTime();
```

Comment:
@Zouxxyy we are trying to reduce flaky tests before we pick up pace on the reviews. Do you think these changes make the test less flaky? if so, love to land this in a minor PR first if possible.

Reply:
Ok, add it, hope it helpful https://github.com/apache/hudi/pull/10526